sdk/appconfiguration/app-configuration/samples/v1/javascript/sample.env

```diff
@@ -0,0 +1,20 @@
+# This is the connection string for your app-configuration resource
+# You can get this from the Azure portal.
+APPCONFIG_CONNECTION_STRING=<app-configuration connection string goes here>
+AZ_CONFIG_ENDPOINT=<ex: https://sample.azconfig.io>
+
+# Register a new application in AAD and assign the "App Configuration Data Owner" role to it
+#   - See https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app
+#     to register a new application in the Azure Active Directory.
+#   - Note down the CLIENT_ID and TENANT_ID from the above step.
+#   - In the "Certificates & Secrets" tab, create a secret and note that down.
+#   - In the Azure portal, go to your App Configuration resource and click on the Access control (IAM)
+#     tab.
+#   - Assign the "App Configuration Data Owner" role to the registered application.
+AZURE_CLIENT_ID=
+AZURE_TENANT_ID=
+AZURE_CLIENT_SECRET=
+
+# Our tests assume that TEST_MODE is "playback" by default. You can
+# change it to "record" to generate new recordings, or "live" to bypass the recorder entirely.
+# TEST_MODE=playback
```

Comment:
Instead of adding a new `test.env` and splitting up the existing `sample.env`, ask for another `sample.env` under the samples folder?

Instead of `package/sample.env` & `package/test.env`, have `package/sample.env` & `package/samples/sample.env`?

Reply:
@witemple-msft

Reply:
As is, only `sample.env` in the package root can be used. Part of the reason I made this change is because `sample.env` is at least lexically tied to the samples directory, so the association between the two (and splitting test-related env settings off into `test.env` being related to the "test" folder) made sense to me. If you feel it's worth it, we could explore some options for using different files, like `samples-dev/sample.env`, or a configurable setting in the `//sampleConfiguration`.

Reply:
I don't mind sample.env and test.env.
I just thought it might be an additional step for the devs to split.
I can live with it.